pages/index.rst

```diff
@@ -8,4 +8,66 @@
 .. type: text
 .. hidetitle: True
 
-.. image:: /images/20190909-eoc-en325.jpg
+
+.. thumbnail:: /images/20190909-eoc-en325.jpg
+  :width: 600
+  :align: center
+  :alt: EOC at the TU Studio
+	
+
+------------------------
```

Comment:
This is basically reimplementing what the RSS posts do atomically.
It makes no sense to copy paste this amount of data on and off the index continuously. This is not how a website works. This is also not how data management works.